composer.json

```diff
@@ -20,7 +20,6 @@
         "symfony/framework-bundle": "~2.5|~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8",
```

Comment:
Except if there is a reason that I don't see, I'd rather leave this dependency :)